static/app/views/insights/navigation.tsx

```diff
@@ -60,7 +63,9 @@ export default function InsightsNavigation({children}: InsightsNavigationProps)
             <SecondaryNav.Item to={`${baseUrl}/${MOBILE_LANDING_SUB_PATH}/`}>
               {MOBILE_SIDEBAR_LABEL}
             </SecondaryNav.Item>
-            <SecondaryNav.Item to={`${baseUrl}/${AI_LANDING_SUB_PATH}/`}>
+            <SecondaryNav.Item
+              to={`${baseUrl}/${AI_LANDING_SUB_PATH}/${MODULE_BASE_URLS[ModuleName.AI]}/`}
```

Comment:
You can also pass `activeTo={`${baseUrl}/${AI_LANDING_SUB_PATH}`}` if you will have other modules in the future to show the nav item as active

Reply:
Roger, thanks! No such plans at this time